src/Pokemon.java

```diff
@@ -0,0 +1,75 @@
+import java.util.Arrays;
+import java.util.List;
+
+public abstract class Pokemon {
+
+    private int level;
+    private int hp;
+    private String food;
+    private String sound;
+    private String name;
+    private String type;
+
+
+    //constructor
```

Comment:
Dit soort comments hoef je niet te pushen. Ik kan wel zien dat het om een constructor, geter of setter gaat